cross/gensio/Makefile

```diff
@@ -0,0 +1,29 @@
+PKG_NAME = gensio
+PKG_VERS = 1.0
+PKG_EXT = tar.gz
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+PKG_DIST_SITE = https://github.com/cminyard/$(PKG_NAME)/archive
+PKG_DIST_NAME = v$(PKG_VERS).$(PKG_EXT)
+
+HOMEPAGE = https://sourceforge.net/projects/ser2net/
+COMMENT  = gensio - General Stream I/O
+LICENSE  = GPLv2
+
+DEPENDS = cross/openssl
+
+GNU_CONFIGURE = 1
+PRE_CONFIGURE_TARGET = myPreConfigure
```

Comment:
please name this label `gensio_pre_configure`
(and remove commented instructions)